packages/block-library/src/footnotes/format.js

```diff
@@ -62,6 +63,16 @@ export const format = {
 			return null;
 		}
 
+		// Checks if the selected block lives within a pattern.
+		if (
+			getBlockParentsByBlockName(
+				getSelectedBlockClientId(),
+				'core/block'
```

Comment:
Minor: Is there any way to check for this without hardcoding the block name?

Reply:
Good question. 

I'm not sure about that, but maybe @glendaviesnz or @aaronrobertshaw might have a better idea about to detect whether a block is inside a pattern? 

Context for those folks: here we want to know to know if any of a block's parents are  patterns by filtering parents using `core/block`.

Reply:
Oh I just discovered `isReusableBlock`

Reply:
> Oh I just discovered isReusableBlock 

Doesn't help here 😄

Reply:
Also, `isReusableBlock` hard codes the `core/block` name as well. 

No alternatives spring to mind but @glendaviesnz has explored this area more so might have better ideas.

Reply:
To-date I have only used `isReusableBlock` sorry, I don't think there is currently an alternative but maybe it is worth adding a new selector along the lines of `isParentSyncedPattern`?

Reply:
>  I don't think there is currently an alternative but maybe it is worth adding a new selector along the lines of isParentSyncedPattern?

Sounds like a useful follow up. Thank you!